sycl/include/sycl/vector_preview.hpp

```diff
@@ -148,18 +129,31 @@ class vec : public detail::vec_arith<DataT, NumElements> {
   // in the class, so vec<float, 16> should be equal to float16 in memory.
   using DataType = std::array<DataT, AdjustedNum>;
 
-public:
 #ifdef __SYCL_DEVICE_ONLY__
+  // clang-format: off
+  using element_type_for_vector_t = typename detail::map_type<
+      DataT,
+#if (!defined(_HAS_STD_BYTE) || _HAS_STD_BYTE != 0)
+      std::byte, /*->*/ std::uint8_t,
+#endif
+      bool, /*->*/ std::int8_t, sycl::half,
```

Comment:
Why the change in formatting? I liked the previous formatting better:
```
bool, /*->*/ std::int8_t,
sycl::half, /*->*/ sycl::detail::half_impl::StorageT,
sycl::ext::oneapi::bfloat16, /*->*/ sycl::ext::oneapi::detail::Bfloat16StorageT,
```

Reply:
Wasn't intentional, uploaded new version.